application/views/admin/token/tokensummary.php

```diff
@@ -4,6 +4,10 @@
  * @var Survey $oSurvey
  * @var array $queries
  */
+
+// DO NOT REMOVE This is for automated testing to validate we see that page
+echo viewHelper::getViewTestTag('surveyParticipantsIndex');
```

Comment:
This one is not visible to phantomjs for some reason. Some ajax thing?

Reply:
Don't know. Check if the web driver can dump the HTML for you.

Reply:
Or you might have to wait until it's visible?